app/models/child.rb

```diff
@@ -1,8 +1,6 @@
 class Child < ApplicationRecord
   include NameIdentifiable
 
-  has_many :placements, inverse_of: :child
```

Comment:
How do we keep an archive of child placements?

Reply:
The placements stay there through `PlacementNeed`. 

In terms of keeping this relationship, we could do that but I don't think that semantically it makes much sense to use. We'll probably need something like `#current_placement` or `#previous_placements`, etc. We can add these as we go.